src/edges_analysis/analysis/beams.py

```diff
@@ -523,12 +521,12 @@ def simulate_spectra(
     lst = np.zeros(72 // twenty_min_per_lst)  # TODO: magic number
 
     antenna_temperature_above_horizon = np.zeros((len(lst), len(beam.frequency)))
-    for i, j, temperature, _, _, _ in _iterate_through_lst_freq(
+    for i, j, temperature, convolution_product, ref_time, _ in _iterate_through_lst_freq(
         lst, ground_loss_file, beam, twenty_min_per_lst, sky_model, index_model, normalize_beam
     ):
         antenna_temperature_above_horizon[i, j] = temperature
 
-    return antenna_temperature_above_horizon, beam.frequency, lst
+    return antenna_temperature_above_horizon, beam.frequency, lst, convolution_product, ref_time
```

Comment:
Adding extra outputs is causing one of the tests to fail. THat can be fixed by changing the call in the test easily enough, but I just wanted to make sure you really want to return these things -- what's the benefit of returning them?

Reply:
These new outputs are needed for our LST binned analysis. Actually I am going to output one more product - sky above horizon list (values at all frequencies and lat/lon)

Reply:
OK, excellent.